controllers/admin/AdminTranslationsController.php

```diff
@@ -1570,6 +1571,8 @@ public function postProcess()
                 } else {
                     $this->errors[] = $this->trans('You do not have permission to edit this.', array(), 'Admin.Notifications.Error');
                 }
+            }elseif (Tools::isSubmit('submitSelectModules')){
+                $this->redirect(false,false,true);
```

Comment:
really, you could use here this code which was moved into `redirect` funtion, i think it would be better :+1: